server/src/test/java/org/elasticsearch/index/mapper/DoubleFieldMapperTests.java

```diff
@@ -52,4 +54,34 @@ protected Number randomNumber() {
          */
         return randomBoolean() ? randomDoubleBetween(-Double.MAX_VALUE, Double.MAX_VALUE, true) : randomFloat();
     }
+
+    public void testScriptAndPrecludedParameters() {
```

Comment:
These were already on the `long` field tests but I think it's a good idea to have them here as well

Reply:
Should they be in `MapperScriptTestCase` then? Like a `final` test method? I think we have a reasonable way to get the `type` being tested, right?

Reply:
They only apply to double and long though.  I'm sure there will be precluded params for date, keyword, etc, but they will be different.  I may end up adding a `getPrecludedParams` method when I add the next implementations though.

Reply:
:+1: to keeping them separate then. I get blind to copy-and-paste differences so I try to avoid them if possible. But, shrug.